dataprofiler/profilers/unstructured_text_profile.py

```diff
@@ -0,0 +1,223 @@
+from collections import defaultdict
+from . import utils, BaseColumnProfiler
+import itertools
+
+class TextProfiler(object):
+    type = 'unstructured_text'
+
+    def __init__(self, name, options=None):
+        """
+        Initialization of Text Profiler.
+
+        :param name: Name of the data
+        :type name: String
+        :param options: Options for the Text Profiler
+        :type options: UnstructuredTextOptions
+        """
+        self.name = name
+        self.sample_size = 0
+        self.times = defaultdict(float)
+        self.vocab = set()
+        self.word_count = defaultdict(int)
+        self.metadata = dict()
```

Comment:
we should consider an option for setting case sensitvity

Reply:
done.